src/main/java/io/github/thebusybiscuit/sensibletoolbox/items/EnderTuner.java

```diff
@@ -81,7 +81,7 @@ public boolean hasGlow() {
 
     @Override
     public void onInteractItem(PlayerInteractEvent event) {
-        if (event.getAction() == Action.RIGHT_CLICK_AIR || event.getAction() == Action.RIGHT_CLICK_BLOCK) {
+        if (event.getAction() == Action.RIGHT_CLICK_AIR || event.getAction() == Action.RIGHT_CLICK_BLOCK) { // Make sure user is interacting intentionally (i.e. Not in a GUI)
```

Comment:
1. Code and comments should never go on the same line
2. The comment is a bit misleading, the `PlayerInteractEvent` is not called for GUI events anyway?

Reply:
Given all other onInteractItem methods from other items have a getAction check before without comment and you can see the issue it's resolving, shall I just remove the comment? I originally shipped without one as it seemed to just bring it in line with other items?

As for why this fires, im not sure if its just the SF Guide, but any GUI icon clicked makes the player interact with their item in hand - so this isn't specific to the endertuner on further inpection. Here is a clip from a server with SF only : 
https://user-images.githubusercontent.com/20646323/124713615-b1224180-def8-11eb-9922-e15a7c5c306b.mp4

I had made the assumption that this behavior was normal and intended, if not, let me know and ill create an issue on the SF GitHub.

Regardless, we still only want the tuner to trigger is the player is using it in a natural manner so limiting to these action types is still wise I think?